config/routes.rb

```diff
@@ -1,7 +1,14 @@
 Rails.application.routes.draw do
   devise_for :users, sign_out_via: [:get, :post]
 
-  
+  namespace :api do
+    resources :users do
+      resources :posts do
```

Comment:
- [ ] The api to create 
`api_user_posts GET      /api/users/:user_id/posts(.:format)` should returns all post in JSON format. However, I got a `404` error and below are the screenshots

Screenshot 1 |
:-----------------:|
<img width="1120" alt="Screen Shot 2022-11-12 at 5 22 12 PM" src="https://user-images.githubusercontent.com/39523312/201486685-ad43eee2-a5b8-4864-84dd-bd37ed1c21ca.png"> |

Screenshot 2 |
:--------------------:|
<img width="1126" alt="Screen Shot 2022-11-12 at 5 22 30 PM" src="https://user-images.githubusercontent.com/39523312/201486692-2dd32663-48ce-4084-a289-49851c987c47.png"> |

 - [ ] Please also check out the endpoint for authentication, both registration and login
    -   Login endpoint - POST : /login
    -   Registration endpoint - POST : /register or /signup